src/library_strings.js

```diff
@@ -226,9 +226,9 @@ mergeInto(LibraryManager.library, {
 
   // Allocate heap space for a JS string, and write it there.
   // It is the responsibility of the caller to free() that memory.
-  $allocateUTF8: function(str) {
+  $stringToNewUTF8: function(str) {
```

Comment:
It would probably be good to collect the contents of runtime_strings.js (including the accompanying `stringToUTF8`) here as well.

Reply:
Yes, that is part of our overall goal of moving things from runtime to library functions..